_data/translations/support.yml

```diff
@@ -0,0 +1,6 @@
+title:
+  en: "Support the call"
+  fr: "Soutenir l’appel"
+subtitle:
+  en: "All States, companies and associations can commit themselves"
+  fr: "Tous les États, entreprises et associations peuvent s’engager"
```

Comment:
This was already translated elsewhere; either share and reuse the translation or update this one to include corrections made in the other.